Xamarin/Droid/Screens/LoginDialog.cs

```diff
@@ -0,0 +1,74 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using DrawXShared;
+
+namespace DrawX.Droid
+{
+    public class LoginDialog : DialogFragment
+    {
+        public Action<bool> OnCloseLogin { get; set; }  // caller should set so can use to dismiss
+
+        public override Dialog OnCreateDialog(Bundle savedInstanceState)
+        {
+            OnCreate(savedInstanceState);
+
+            // equivalent of an Activity's SetContentView
+            var builder = new AlertDialog.Builder(Activity);
+            var inflator = Activity.LayoutInflater;
```

Comment:
Not sure what's the point of extracting that to a new `var`. Combining with the next line seems simpler